### ISSUE TITLE
New plugin welcome?

### ISSUE THREAD

**Issue by suntong** [songtianyi/wechat-go#34]
I'm planing to write a new plugin for your excellent package. 
Would you accept PR for it? 

I can provide more details if interested. 

thx

**Comment by songtianyi** [comment]
Sure, any contributions would be appreciated.

**Comment by suntong** [comment]
Thanks I'll see...

**Comment by suntong** [comment]
Done -- it is at https://github.com/suntong/wx-go/tree/master/plugins/gamer24

I figure I'd better to keep it in a separated repo than submitting PR each time I need to amend it. 
and that idea turns out to be useful as I'm having problem make my [first `wechat-go` application](https://github.com/suntong/wx-go/tree/master/cmd/wx-go) work.

I found that whenever I get that specific command "3824", immediately I'll get `[I] trying re-login with cache` from the console log. Tried many time and it is always the same. 

So I added logging in my plugin, but none of the loggings are printed to the console, so I suspect that something else is wrong. 

Would you look into my  [simple `wechat-go` application](https://github.com/suntong/wx-go/tree/master/cmd/wx-go) and see what went wrong please?

Thanks a lot!

**Comment by songtianyi** [comment]
Nice plugin!
I will check the code when i'm free.

**Comment by songtianyi** [comment]
Again, it works for my account :(
``` 
5 4
 8 8
-!!
:(5 - 8 / 4) * 8

 6 8
 8 9
-!!
:8 * 9 - 6 * 8
 3 8
 6 5
-!!
:3 * 8 * (6 - 5)

 1 4
 1 6
-!!
:(1 + 4 - 1) * 6
```

**Comment by songtianyi** [comment]
How to play this game in wechat?

**Comment by suntong** [comment]
- just scroll back and reveal only the first four cards on the mobile device
- If you don't think there is an answer, scroll one more line to reveal the hint.
  * `-!!` means there is a solution. 
  * `-XX` means there is indeed no solution.
- The solution given is the fist one found, not mean to cover all cases.

**Comment by suntong** [comment]
So you just build and run my wx-go as is, or you added the plugin to your own code?

**Comment by songtianyi** [comment]
> So you just build and run my wx-go as is, or you added the plugin to your own code?

run you wx-go as it is.

**Comment by suntong** [comment]
k. thx. will try again.

**Comment by suntong** [comment]
Hey, sorry to get back late to you. 

I think I know what's wrong -- I've always been scanning-in on the second attempt. I.e., after I saw the following time out error in my console. 

2017/12/20 19:32:01 [W] login response, window.code=408;
2017/12/20 19:32:01 [E] session exit, login response, window.code=408;

whereas you must have been always scan-in right away. Today I did that, and everything works out for me this time. I'm opening another thread to talk about it in details -- https://github.com/songtianyi/wechat-go/issues/39.
